src/js/modules/users/components/ContributorPage.js

```diff
@@ -1,49 +1,85 @@
 import React from "react";
 import { connect } from "react-redux";
-import { Link } from "react-router-dom";
 import injectSheet from "react-jss";
+import { Grid, Row, Col } from "react-bootstrap/lib";
 
 import { getContributorFromSlug } from "../selectors";
 import { getArticlesByContributor } from "../../articles/selectors";
 import { getSections } from "../../sections/selectors";
+import { ArticleRow } from "../../articles/components";
```

Comment:
Why are you importing an ArticleRow? Just use ArticleList. ArticleLists are containers for ArticleRow, and all you have to do is feed the ArticleList its props (listed in ArticleList.js).